R-package/R/lgb.Booster.R

```diff
@@ -469,8 +480,19 @@ Booster <- R6::R6Class(
                        predcontrib = FALSE,
                        header = FALSE,
                        reshape = FALSE,
+                       params = list(),
```

Comment:
I suppose we should update all our tests (and examples, but if I'm not mistaken, we don't have any yet) and use `params` in them to silence new warning.

Reply:
Have you seen any such warnings or places where prediction parameters are passed through in our examples or tests? I checked through the tests and didn't find any uses of extra parameters that didn't match the other keyword arguments already.

To be clear, this would only apply for passing the parameters listed at https://lightgbm.readthedocs.io/en/latest/Parameters.html#predict-parameters, such as `pred_early_stop`.

I searched for those parameters with `git grep` and didn't find any uses in the R package. For example, with 

```shell
git grep pred_early_stop R-package/
```

Reply:
Sorry, didn't know that you had already done it! Then everything is OK with this PR I think. But it's a good sign to add a test for those parameters in R-package 😉

Reply:
No problem, I should have mentioned it. And definitely agree that it's a sign of a missing test.